src/server/shared/Realm/Realm.cpp

```diff
@@ -42,8 +42,8 @@ ip::tcp::endpoint Realm::GetAddressForClient(ip::address const& clientAddr) cons
         {
             realmIp = LocalAddress;
         }
-        else
-            realmIp = ExternalAddress;
+        else 
```

Comment:
Is the extra space needed at the end of these 2 lines? Or could they stay as they were?